### PR TITLE
Issue 239 - API for profile indicators to provide last updated date

### PR DESCRIPTION
**Pull request by Faysalali534**
## Description
The ProfileIndicator API endpoint should include the last_updated date.
this indicator. and maybe more, but let's start here.

## Related Issue
the issue is related to this [link](https://github.com/OpenUpSA/wazimap-ng/issues/239)
## How to test it locally
`sudo docker-compose run --rm -e DJANGO_CONFIGURATION=Test web pytest -s /app/tests/profile/test_views.py::test_profile_last_update`
## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out

### Commits

- [ ]  commits are clean
- [ ]  commit messages are clean

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers
- [ ]  black was run locally (as part of the pre-commit hook)

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

**Review by milafrerichs — CHANGES_REQUESTED** [review]
the profileIndicator already has created and updated via the `BaseModel` 
there is no need to create new fields. please use the existing fields.

File: wazimap_ng/profile/serializers/profile_indicator_serializer.py [hunk 1]
```diff
@@ -6,22 +6,27 @@
...
 
-    class Meta:
-        model = ProfileIndicator
-        exclude = ["profile", "id"]
-        depth = 2
+  def get_last_updated(self, obj):
+      return obj.updated
+
+  class Meta:
+      model = models.ProfileIndicator
```
Comment (milafrerichs) [inline wazimap_ng/profile/serializers/profile_indicator_serializer.py:27]: why are you using `model.ProfileIndicator` when `ProfileIndicator` is already imported.

File: tests/profile/test_views.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    return OrderedDict([
+           ('15-19',OrderedDict(
+                            M={"count": 8.79722},
+                            F={"count": 9.62006},
+                    )),
+            ('20-24',OrderedDict(F={"count": 2.2397}))
+        ])
+
+@pytest.mark.django_db
+class TestProfileGeographyData:
```
Comment (milafrerichs) [inline tests/profile/test_views.py:55]: why are those tests in your changes? they have nothing to do with your change, right?

**Review by milafrerichs — CHANGES_REQUESTED** [review]
the tests are failing. please fix them.

**Comment by Faysalali534** [comment]
Yes have fixed it. Fixing conflicts.
